addons/sale_project/controllers/portal.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from operator import itemgetter
+
+from odoo import http, _
```

Comment:
Both this imports are not more needed since the route override has  been removed 🙂